### ISSUE TITLE
Player Bank does not show contents of Bank Bag 7

### ISSUE THREAD

**Issue by alphabitnz** [tullamods/Bagnon#1463]
Seems like Blizz might have offset bank bag numbers by 1, Player Bank Bag 7 no longer being found by Bagnon.
Had to use workaround for GetBankBagSlotFlag as per AtreyuGreen's comment in #1461 

![image](https://user-images.githubusercontent.com/9102111/198858833-541395c5-144b-4480-9ef4-1bcb5ca6864f.png)

**Comment by Salmanius** [comment]
Yes it does seem to be an issue with the numbering being slightly different. 
Interface\AddOns\Bagnon\common\Wildpants\classes\bank.lua:12
`for slot = 1, NUM_BANKBAGSLOTS do
	tinsert(Bank.Bags, slot + NUM_BAG_SLOTS+1)
end`

This (mostly) fixes the bank and shows all bags.

**Comment by alphabitnz** [comment]
> Yes it does seem to be an issue with the numbering being slightly different. Interface\AddOns\Bagnon\common\Wildpants\classes\bank.lua:12 `for slot = 1, NUM_BANKBAGSLOTS do tinsert(Bank.Bags, slot + NUM_BAG_SLOTS+1) end`
> 
> This (mostly) fixes the bank and shows all bags.

Brilliant! Can confirm this fixes the bank bag 7 issue

**Comment by Salmanius** [comment]
After a bit more messing, I think I have a solid fix. I pull requested this to the authors but until they accept or implement it themselves another way, you can see the changes necessary here: 
https://github.com/tullamods/Wildpants/pull/85

**Comment by WanderingFox** [comment]
@Hinos you need to make a similar change in BagBrother too

Change these two lines at the top of `BagBrother/Code/Events.lua`

```
local FIRST_BANK_SLOT = 1 + NUM_BAG_SLOTS
local LAST_BANK_SLOT = NUM_BANKBAGSLOTS + NUM_BAG_SLOTS 
```

to

```
local FIRST_BANK_SLOT = 1 + NUM_BAG_SLOTS + 1
local LAST_BANK_SLOT = NUM_BANKBAGSLOTS + NUM_BAG_SLOTS + 1
```

tl;dr: blizz added another bag slot but never incremented `NUM_BAG_SLOTS` so it needs a slight adjustment.
